sys/include/ztimer.h

```diff
@@ -337,7 +337,13 @@ struct ztimer_clock {
     ztimer_now_t checkpoint;        /**< cumulated time at last now() call  */
 #endif
 #if MODULE_PM_LAYERED || DOXYGEN
-    uint8_t required_pm_mode;       /**< min. pm mode required for the clock to run */
+    union {
+        uint8_t required_pm_mode;       /**< min. pm mode required for the clock to run
+                                         *   @deprecated name change -> block_pm_mode:
+                                         *               this is used to block a pm_mode
+                                         *               use block_pm_mode instead */
+        uint8_t block_pm_mode;          /**< min. pm mode to block for the clock to run */
+    };
```

Comment:
Do we really consider this a user facing API that needs deprecation? IMO this is internal and we should just rename it and be done with it.

Reply:
Fine with me

Reply:
so @kfessel, sorry for the confusion, please just remove the deprecation stuff...

Reply:
I'm also fine with just renaming it. This struct member is just touched by `auto_init.c`. At least in all my use cases ;)

Reply:
done